.github/workflows/causal_pipeline_sim_outcomes.yml

```
@@ -91,6 +91,19 @@ jobs:
           source .github/workflows/pipeline_helpers.sh
           
           run_step main_causal.estimate        estimate_simulated_with_bias.yaml
+      - name: Run XGBoost baseline test
+        shell: bash
+        run: |
+          set -euo pipefail
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)